terps/scott/gameinfo.c

```diff
@@ -0,0 +1,188 @@
+//
+//  gameinfo.c
+//  scott
+//
+//  Created by Administrator on 2022-01-30.
+//
+
+#include <stdio.h>
+
+#include "definitions.h"
+
+/* This is supposed to be the original ScottFree system
+ messages in second person, as far as possible */
+char *sysdict[MAX_SYSMESS] = {
```

Comment:
Presumably this would be better as `const char *`.

Reply:
Well, I know I'm being annoying here, but there are lots of places that ought to be const throughout the code. Anything which is just string literals should be stored in `const char*`, and when those changes are made, warnings will radiate out to various other functions/variables, but it's simple enough to fix as you find them.

The Glk API, unfortunately, takes some `char *` values when it should take `const char *`, so casts will be required there, but throughout ScottFree things ought to be const except where they're modified.

Reply:
Definitely not annoying! Thanks the review. I hope I got all of them.